ext/fileinfo/tests/bug61964-mb.phpt

```diff
@@ -10,7 +10,7 @@ $magic_file = __DIR__ . DIRECTORY_SEPARATOR . 'magic私はガラスを食べら
 $ret = @finfo_open(FILEINFO_NONE, $magic_file . ".non-exits私はガラスを食べられます");
 var_dump($ret);
 
-$dir = __DIR__ . "/test-folder";
+$dir = __DIR__ . "/.tmp-bug61964-mb";
```

Comment:
By convention we just use the name of the test for temporary directories/files, so just `/bug61964-mb` here.

Reply:
Renamed.